### PR TITLE
Runtime lint: Warning about class or data-bem in `attrs`

### PR DESCRIPTION
**Pull request by miripiruni**
Fix for #212

### CODE REVIEW

File: runtime-lint/index.js [hunk 1]
```diff
@@ -63,3 +63,27 @@ block('*').def()(function() {
...
 });
+
+block('*').match(function() { return this.ctx.attrs; }).def()(function() {
+  var ctx = this.ctx;
+  var attrs = ctx.attrs;
+
+  if (attrs.class) {
+      console.warn(
+        '\nBEM-XJST WARNING: looks like you try to set HTML class from attrs field in BEMJSON. ' +
+        'Please you cls() mode for it. See documentation: https://github.com/bem/bem-xjst/blob/master/docs/en/5-templates-syntax.md#cls' +
```
Comment (tadatuta) [inline runtime-lint/index.js]: you -> use

File: runtime-lint/index.js [hunk 2]
```diff
@@ -63,3 +63,27 @@ block('*').def()(function() {
...
   return ret;
 });
+
+block('*').match(function() { return this.ctx.attrs; }).def()(function() {
+  var ctx = this.ctx;
+  var attrs = ctx.attrs;
+
+  if (attrs.class) {
+      console.warn(
+        '\nBEM-XJST WARNING: looks like you try to set HTML class from attrs field in BEMJSON. ' +
```
Comment (tadatuta) [inline runtime-lint/index.js]: you're trying

File: runtime-lint/index.js [hunk 3]
```diff
@@ -63,3 +63,27 @@ block('*').def()(function() {
...
+        'Please you cls() mode for it. See documentation: https://github.com/bem/bem-xjst/blob/master/docs/en/5-templates-syntax.md#cls' +
+        '\n' + JSON.stringify(ctx)
+      );
+  }
+
+
+  if (attrs['data-bem']) {
+      console.warn(
+        '\nBEM-XJST WARNING: looks like you try to set data-bem attribute from attrs field in BEMJSON. ' +
```
Comment (tadatuta) [inline runtime-lint/index.js]: you're trying

**Comment by miripiruni** [comment]
Check `attrs()`…

File: runtime-lint/index.js [hunk 4]
```diff
@@ -63,3 +63,27 @@ block('*').def()(function() {
...
+        'Please you js() mode for it. See documentation: https://github.com/bem/bem-xjst/blob/master/docs/en/5-templates-syntax.md#js' +
```
Comment (tadatuta) [inline runtime-lint/index.js]: you -> use

File: runtime-lint/index.js (hunk 2 above)
Comment (miripiruni) [inline runtime-lint/index.js]: Fixed

File: runtime-lint/index.js (hunk 1 above)
Comment (miripiruni) [inline runtime-lint/index.js]: Fixed

File: runtime-lint/index.js (hunk 3 above)
Comment (miripiruni) [inline runtime-lint/index.js]: Fixed

File: runtime-lint/index.js (hunk 4 above)
Comment (miripiruni) [inline runtime-lint/index.js]: Fixed

**Comment by miripiruni** [comment]
Updated

**Pull request merged**